package.json

```diff
@@ -22,6 +22,7 @@
     "object-describer": "https://github.com/kubernetes-ui/object-describer.git#v1.0.4",
     "patternfly": "3.27.7",
     "patternfly-bootstrap-combobox": "1.1.7",
+    "patternfly-react": "0.24.0",
```

Comment:
I guess this is blocked until we have a release of `patternfly-react` which contains the `SerialConsole` component.

Reply:
Right. At least `0.25.0` will be required here, but let's see.